### PR TITLE
cleanup: lint and protect against skipped tests

### PR DESCRIPTION
**Pull request by stephanwlee**
As a developer, it is super easy to erroneously commit fdescribe or
fit. This lint protects against that.

### CODE REVIEW

**Comment by caisq** [comment]
driveby: Thanks for doing this, @stephanwlee

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -92,6 +92,9 @@ jobs:
       - run: yarn install --ignore-engines
         # You can run `yarn fix-lint` to fix all Prettier complaints.
       - run: yarn lint
+        # Make sure no tests are skipped with "focused" tests.
+      - run: |
+          ! find tensorboard/webapp -type f -regex '.*_test.ts$' -exec grep -E 'f(it|describe)\(' {} +
```
Comment (wchargin) [inline .github/workflows/ci.yml:97]: This is correct and works fine, but using `git grep` is clearer:

```
! git grep -E 'f(it|describe)\(' 'tensorboard/webapp/*_test.ts'
```

(It’s also often faster, for what it’s worth.)

Comment (caisq) [reply .github/workflows/ci.yml:97]: @stephanwlee , if you go with @wchargin's suggestion, please consider including *_test.ts files under tensorboard/plugins as well.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ jobs:
       - run: yarn lint
         # Make sure no tests are skipped with "focused" tests.
       - run: |
-          ! find tensorboard/webapp -type f -regex '.*_test.ts$' -exec grep -E 'f(it|describe)\(' {} +
+          ! git grep -E 'f(it|describe)\(' 'tensorboard/webapp/*_test.ts' 'tensorboard/plugins/*_test.ts'
```
Comment (wchargin) [inline .github/workflows/ci.yml:97]: Fine with me if this is what you want, though if we’re cherry-picking
two directories it’s a bit unclear to me why we don’t just run against
all of `*_test.ts`.

**Pull request merged**